packages/cloud-cognitive/src/components/TagSet/TagSet.js

```diff
@@ -304,6 +297,15 @@ TagSet.propTypes = {
    * label for the overflow show all tags link. **Note: Required if more than 10 tags**
    */
   showAllTagsLabel: string_required_if_more_than_10_tags,
+  /**
+   * An array of Carbon objects containing Carbon Tag properties plus 'label'. NOTE: 'filter' is not ignored.
```

Comment:
I'm not sure that a "Carbon object" makes sense, and I think 'label' needs just a soupçon more explanation. I've adapted the following from the docs for 'actions' on ActionSet. I didn't include the note about 'filter' because I didn't understand it: see next comment.

```suggestion
   * The tags to be shown in the TagSet. Each tag is specified as an object
   * with optional fields: 'label' to supply the tag label, and additional
   * fields in the object will be passed to the Carbon Tag component: these
   * can include 'type', 'filter', 'disabled', 'ref', 'className', and any
   * other Tag props. Any other fields in the object will be passed through
   * to the HTML element as HTML attributes.
   *
   * See https://react.carbondesignsystem.com/?path=/docs/components-tag--default
```